app/controllers/api/v1/rooms_controller.rb

```diff
@@ -6,6 +6,10 @@ def available
         render json: RoomResRoom.available_rooms(@start_time)
       end
 
+      def available_count
```

Comment:
I don't have any objections to the new API call in general.. but it might be cleaner to just call the available rooms api and just call `.length` on the result. This eliminates the need for extra tests and functionality on the server side.